cloud/gcpprovider.go

```diff
@@ -17,6 +17,7 @@ import (
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	compute "google.golang.org/api/compute/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Similar warning about the `v1` already being `v1` and not requiring the alias

Reply:
Just left for consistency with metav1